src/three-components/ModelScene.js

```diff
@@ -199,62 +174,36 @@ export default class ModelScene extends Scene {
     if (width !== this.width || height !== this.height) {
       this.width = Math.max(width, 1);
       this.height = Math.max(height, 1);
-      this.aspect = this.width / this.height;
       // In practice, invocations of setSize are throttled at the element level,
       // so no need to throttle here:
-      this.applyRoomSize();
+      this.updateFraming();
     }
   }
 
   /**
-   * Updates the 3D room and model scale based on the 2D
-   * dimensions for the encapsulating element.
+   * To frame the scene, a box is fit around the model such that the X and Z
+   * dimensions (modelDepth) are the same (for Y-rotation) and the X/Y ratio is
+   * the aspect ratio of the canvas (framedHeight is the Y dimension). At the
+   * ideal distance, the camera's fov exactly covers the front face of this box
+   * when looking down the Z-axis.
```

Comment:
Yeah, I wanted to call it out because I wasn't sure if everyone was clear on how our framing really worked. It means that for a model that touches its bounding box corners, it will still get clipped at certain rotations. I'm not sure I'd recommend changing the framing, but we might want to allow a little extra zoom out to ensure that any model can be entirely visible in frame from any angle.

Reply:
You can see what I'm talking about here: https://maps3d.io/blog/what-is-the-gltf-glb-format-3d-file/

Reply:
Yeah, I agree with you regarding the extra zoom-out by default. We also want to just let users configure the allowed min/max https://github.com/GoogleWebComponents/model-viewer/issues/458